Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile-upstream:1.4
 
 ARG RUNC_VERSION=v1.0.2
```

Comment:
We should update runc too, but that can be another PR

Reply:
Yes I was talking with @thaJeztah and @tonistiigi about this one. We could temporarily patch libseccomp using `go mod edit` with https://github.com/seccomp/libseccomp-golang/pull/85 in the Dockerfile. WDYT?

Reply:
Can we just download the runc binary from https://github.com/opencontainers/runc/releases/tag/v1.1.1 ?

Reply:
I'd love to see it merged in libseccomp; carried tonis's PR last week to try to unblock it; please review that one (hopefully it will help getting attention of the maintainers there) https://github.com/seccomp/libseccomp-golang/pull/90

Reply:
PR for upstream `runc.riscv64`:
- https://github.com/opencontainers/runc/pull/3463